src/system/applications/combat/combat-tracker.ts

```diff
@@ -0,0 +1,151 @@
+import CosmereCombatant from './combatant';
+
+export default class CosmereCombatTracker extends CombatTracker {
+    //overrides default tracker template to implement slow/fast buckets and activation button.
+    //eslint-disable-next-line -- eslint rules want readonly field
```

Comment:
When using `eslint-disable-next-line` you can specify which rule to disable. While it may not have any further effect here, we should make sure to only disable specific rules when its needed.